ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -0,0 +1,260 @@
+def replace(file, operatorImage, e2eImage, testApiserverImage) {
+	def SRC_E2E_FILE_CONTENT = readFile file: file
+	def DST_E2E_FILE_CONTENT = SRC_E2E_FILE_CONTENT.replaceAll("- --operator-image=.*", "- --operator-image=${operatorImage}")
+	DST_E2E_FILE_CONTENT = DST_E2E_FILE_CONTENT.replaceAll("image:.*", "image: ${e2eImage}")
+	DST_E2E_FILE_CONTENT = DST_E2E_FILE_CONTENT.replaceAll("- --test-apiserver-image=.*", "- --test-apiserver-image=${testApiserverImage}")
+	DST_E2E_FILE_CONTENT = DST_E2E_FILE_CONTENT.replaceAll("imagePullPolicy: Always", "imagePullPolicy: IfNotPresent")
+	writeFile file: file, text: "${DST_E2E_FILE_CONTENT}"
+}
+
+def replace_wh(file, operatorImage) {
+	def SRC_WH_FILE_CONTENT = readFile file: file
+	def DST_WH_FILE_CONTENT = SRC_WH_FILE_CONTENT.replaceAll("image:.*", "image: ${operatorImage}")
+	writeFile file: file, text: "${DST_WH_FILE_CONTENT}"
+}
+
+def getChangeLogText() {
+	def changeLogText = ""
+	for (int i = 0; i < currentBuild.changeSets.size(); i++) {
+		for (int j = 0; j < currentBuild.changeSets[i].items.length; j++) {
+			def commitId = "${currentBuild.changeSets[i].items[j].commitId}"
+			def commitMsg = "${currentBuild.changeSets[i].items[j].msg}"
+			changeLogText += "\n" + "`${commitId.take(7)}` ${commitMsg}"
+		}
+	}
+	return changeLogText
+}
+
+def call(BUILD_BRANCH, CREDENTIALS_ID) {
+
+	def GITHASH
+	def UCLOUD_OSS_URL = "http://pingcap-dev.hk.ufileos.com"
+	def BUILD_URL = "git@github.com:pingcap/tidb-operator.git"
+	def PROJECT_DIR = "go/src/github.com/pingcap/tidb-operator"
+
+	catchError {
+		node('build_go1130_memvolume'){
+			container("golang") {
+				def WORKSPACE = pwd()
+				dir("${PROJECT_DIR}"){
+					stage('build tidb-operator binary'){
+						checkout changelog: false, poll: false, scm: [$class: 'GitSCM', branches: [[name: "${BUILD_BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: "${CREDENTIALS_ID}", refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: "${BUILD_URL}"]]]
+						//git credentialsId: "k8s", url: "${BUILD_URL}", branch: "${ghprbActualCommit}"
+						GITHASH = sh(returnStdout: true, script: "git rev-parse HEAD").trim()
+						sh """
+						export GOPATH=${WORKSPACE}/go
+						export PATH=${WORKSPACE}/go/bin:\$PATH
+						if ! hash hg 2>/dev/null; then
+							sudo yum install -y mercurial
+						fi
+						hg --version
+						make
+						rm -rf tests/images/test-apiserver/bin
+						make e2e-build
+						"""
```

Comment:
must run `make test GO_COVER=y` and report the results to `https://codecov.io` on master branch

note that I guess we should use the token in Jenkins credentials.

Reply:
or we can run it in github actions instead?

Reply:
cc @weekface @tennix what do you think?

Reply:
I think it's okay to add this logic to this script.